CarpetX/src/schedule.cxx

```diff
@@ -1023,6 +1025,33 @@ std::vector<clause_t> decode_clauses(const cFunctionData *restrict attribute,
   return result;
 }
 
+CCTK_REAL get_coarse_mindx() {
+  CCTK_REAL mindx = std::numeric_limits<CCTK_REAL>::infinity();
+  for (const auto &patchdata : ghext->patchdata) {
+    const amrex::Geometry &geom = patchdata.amrcore->Geom(0);
+    const CCTK_REAL *restrict const dx = geom.CellSize();
+    CCTK_REAL mindx1 = std::numeric_limits<CCTK_REAL>::infinity();
+    for (int d = 0; d < dim; ++d)
+      mindx1 = fmin(mindx1, dx[d]);
+    mindx = fmin(mindx, mindx1);
+  }
+  return mindx;
+}
+
+CCTK_REAL get_finest_mindx() {
+  CCTK_REAL mindx = std::numeric_limits<CCTK_REAL>::infinity();
+  for (const auto &patchdata : ghext->patchdata) {
+    const amrex::Geometry &geom = patchdata.amrcore->Geom(0);
+    const CCTK_REAL *restrict const dx = geom.CellSize();
+    CCTK_REAL mindx1 = std::numeric_limits<CCTK_REAL>::infinity();
+    for (int d = 0; d < dim; ++d)
+      mindx1 = fmin(mindx1, dx[d]);
+    mindx1 = ldexp(mindx1, -(int(patchdata.leveldata.size()) - 1));
```

Comment:
`ldexp` seems to be a somewhat obscure function. Not sure of the benefit of efficiency here is worth the non-intuitive name. This was a `pow(2, -n)` before I guess? Since CarpetX already requires Arith, why not use its `pown`?